pkg/client/multi.go

```diff
@@ -408,12 +409,37 @@ func (m *MultiClient) ProjectList(ctx context.Context) ([]apiv1.Project, error)
 	})
 }
 
-func (m *MultiClient) Info(ctx context.Context) (*apiv1.Info, error) {
-	c, err := m.Factory.ForProject(ctx, m.Factory.DefaultProject())
+func (m *MultiClient) Info(ctx context.Context) ([]apiv1.Info, error) {
+	var projectsInfo []apiv1.Info
+	// Can this be refactored into a call to aggregate?
+	projects, err := m.ProjectList(ctx)
 	if err != nil {
 		return nil, err
 	}
-	return c.Info(ctx)
+	var subprojectInfo []apiv1.Info
+	// For each project after listing them all, call info on each and populate projectsInfo slice
+	for _, project := range projects {
+		c, err := m.Factory.ForProject(ctx, project.Name)
+		if err != nil {
+			return nil, err
+		}
+		subprojectInfo, err = c.Info(ctx)
+		if err != nil {
+			return nil, err
+		}
+		if len(subprojectInfo) == 1 {
```

Comment:
It would be cleaner to just do 

```
for _, info := range subprojectInfo {
  info.name = project.name
  projectsInfo = append(projectsInfo, info)
}
```

So if a project can't list info, I fee like it is better UX to users that we just silently ignore it instead of throwing an error.